### PR TITLE
add webidl as a possible deliverable

### PR DESCRIPTION
**Pull request by siusin**
To address comments from #49 and [strategy#269](https://github.com/w3c/strategy/issues/269#issuecomment-854072755).

- Add WebIDL to the Deliverables section;
- Remove the WebSec IG from Coordination;
- Switch to the latest charter template for the  Success Criteria section and the Coordination section;

### CODE REVIEW

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
(holding off on merging until others have reviewed)

File: charter/draft-charter-2021.html
Comment (plehegar) [inline charter/draft-charter-2021.html:223]:
```suggestion
            "https://www.w3.org/Consortium/Process/#rec-pr">Proposed
```

File: charter/draft-charter-2021.html
Comment (plehegar) [inline charter/draft-charter-2021.html:409]:
```suggestion
            "https://www.w3.org/Consortium/Process/#Consensus">Consensus</a> of the Working Group members, the Group may bring the following WHATWG Review Drafts from W3C Candidate Recommendation to Proposed Recommendation, in accordance with the <a href="https://www.w3.org/2021/03/WHATWG-W3C-MOU.html">WHATWG-W3C Memorandum of Understanding</a>: <!-- @@update with latest MOU link once available -->
```

**Review by plehegar — CHANGES_REQUESTED** [review]

**Review by marcoscaceres — APPROVED** [review]
With @plehegar suggestions, lgtm.

**Comment by marcoscaceres** [comment]
@siusin, before merging, could you kindly please run:

```
tidy -config tidyconfig.txt -o charter/draft-charter-2021.html charter/draft-charter-2021.html
```

Thanks 🙏

**Comment by siusin** [comment]
@marcoscaceres @plehegar 
Done. Thanks!

**Review by plehegar — APPROVED** [review]

**Pull request merged**
